src/LndClient/Data/OpenChannel.hs

```diff
@@ -26,10 +26,10 @@ instance ToGrpc OpenChannelRequest GRPC.OpenChannelRequest where
   toGrpc x =
     msg
       <$> toGrpc (nodePubkey x)
-      <*> toGrpc (localFundingAmount x)
-      <*> toGrpc (pushSat x)
+      <*> (toGrpc =<< toSat (localFundingAmount x))
+      <*> maybe (Right def) (toGrpc <=< toSat) (pushSat x)
```

Comment:
What is `def`? Default protobuf value?

Reply:
Yes

Reply:
done

Reply:
Maybe it's wrong to set this value to default (which is `0`)?

Reply:
Oh, I probably got it wrong - it sets `def` in case of `Nothing`, and fails as `Left` in case of `toGrpc` failure?

Reply:
Yep, if Nothing it will set default value, if Some - it will transform it.

Reply:
So transformation could fail